i18n/management/commands/i18n_sync_up.py

```diff
@@ -14,21 +15,28 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         log("I18n Sync Step 2 of 4: Upload source strings to Crowdin")
         source_dir = os.path.join(I18nFileWrapper.static_dir(), 'source')
-        redacted_dir = os.path.join(I18nFileWrapper.static_dir(), 'redacted')
+        upload_dir = os.path.join(I18nFileWrapper.static_dir(), 'upload')
 
-        if not os.path.exists(redacted_dir):
-            os.makedirs(redacted_dir)
+        # Strings uploaded should include all source strings, some of which
+        # will need to be redacted.
+        copy_tree(source_dir, upload_dir)
```

Comment:
Why is this necessary?

Reply:
We need the `upload` directory to contain all the stuff that gets uploaded to crowdin. That includes redacted content, so we _also_ need somewhere to keep unredacted ("source") content so we can restore the translations.

Does that answer your question?

Reply:
Makes sense. Thanks!